fe/fe-core/src/main/java/org/apache/doris/nereids/operators/plans/logical/LogicalAggregation.java

```diff
@@ -31,68 +32,70 @@
 
 /**
  * Logical Aggregation plan operator.
- *
- *eg:select a, sum(b), c from table group by a, c;
+ * <p>
+ * eg:select a, sum(b), c from table group by a, c;
  * groupByExpressions: Column field after group by. eg: a, c;
  * outputExpressions: Column field after select. eg: a, sum(b), c;
- *
+ * <p>
  * Each agg node only contains the select statement field of the same layer,
  * and other agg nodes in the subquery contain.
  */
 public class LogicalAggregation extends LogicalUnaryOperator {
 
-    private final List<Expression> groupByExpressions;
-    private final List<? extends NamedExpression> outputExpressions;
+    private final List<Expression> groupByExprList;
+    private final List<NamedExpression> aggExprList;
+    private List<Expression> partitionExprList;
+
+    private AggPhase aggPhase;
 
     /**
      * Desc: Constructor for LogicalAggregation.
      */
-    public LogicalAggregation(List<Expression> groupByExpressions,
-            List<? extends NamedExpression> outputExpressions) {
+    public LogicalAggregation(List<Expression> groupByExprList, List<NamedExpression> aggExprList) {
         super(OperatorType.LOGICAL_AGGREGATION);
-        this.groupByExpressions = groupByExpressions;
-        this.outputExpressions = outputExpressions;
+        this.groupByExprList = groupByExprList;
+        this.aggExprList = aggExprList;
     }
 
-    /**
-     * Get GroupByAggregation list.
-     *
-     * @return all group by of this node.
-     */
-    public List<Expression> getGroupByExpressions() {
-        return groupByExpressions;
+    public List<Expression> getPartitionExprList() {
+        return partitionExprList;
     }
 
-    /**
-     * Get outputExpressions list.
-     *
-     * @return all agg expressions.
-     */
-    public List<? extends NamedExpression> getoutputExpressions() {
-        return outputExpressions;
+    public void setPartitionExprList(List<Expression> partitionExprList) {
+        this.partitionExprList = partitionExprList;
+    }
+
+    public List<Expression> getGroupByExprList() {
+        return groupByExprList;
+    }
+
+    public List<NamedExpression> getAggExprList() {
+        return aggExprList;
+    }
+
+    public AggPhase getAggPhase() {
+        return aggPhase;
     }
 
     @Override
     public String toString() {
-        return "Aggregation (" + "outputExpressions: " + StringUtils.join(outputExpressions, ", ")
-                + ", groupByExpressions: " + StringUtils.join(groupByExpressions, ", ") + ")";
+        return "Aggregation (" + "outputExpressions: " + StringUtils.join(aggExprList, ", ") + ", groupByExpressions: "
```

Comment:
Is it better to unify it with the variable name here？outputExpressions=>aggExprList, etc.